readme.md

```diff
@@ -84,21 +93,37 @@ The following are valid Decimals:
 let a = 1.0;
 let b = 0.5;
 let c = 100.25;
+let d = 0.0;
+```
+
+The following are vaild negative Numbers:
+```
+let a = -1;
+let b = -0.5;
+let c = -0;
 ```
 
+
 Trailing or leading decimals on numbers are invalid and should be detected as such by the compiler. Because of this, the following are **invalid**
 ```
 let a = .1;
 let b = 5.;
 ```
 
+Putting the minus sign in the back is invalid. Because of this and the previous rule, the following are **invalid**
```

Comment:
Could probably be worded to explain that putting a - sign there as a subtraction is fine, for instance 0-5 would be -5